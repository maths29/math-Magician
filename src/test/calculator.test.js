import React from 'react';
import { render } from '@testing-library/react';
import Calculator from '../components/calculator';

test('Calculator renders correctly on the browser', () => {
  const { asFragment } = render(<Calculator />);
  expect(asFragment()).toMatchSnapshot();
});
