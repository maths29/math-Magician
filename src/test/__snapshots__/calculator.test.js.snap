// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly on the browser 1`] = `
<DocumentFragment>
  <div>
    <div
      class="prompAndCalDiv"
    >
      <h2
        class="myh2"
      >
        Let's do some math!
      </h2>
      <div
        class="calInterface"
      >
        <div
          class="calculator"
        >
          <div
            class="buttonDiv"
          >
            <div
              class="inputField"
            >
              0
            </div>
          </div>
          <div
            class="buttonDiv"
          >
            <button
              type="button"
            >
              AC
            </button>
            <button
              type="button"
            >
              +/-
            </button>
            <button
              type="button"
            >
              %
            </button>
            <button
              class="operation"
              type="button"
            >
              รท
            </button>
          </div>
          <div
            class="buttonDiv"
          >
            <button
              type="button"
            >
              7
            </button>
            <button
              type="button"
            >
              8
            </button>
            <button
              type="button"
            >
              9
            </button>
            <button
              class="operation"
              type="button"
            >
              x
            </button>
          </div>
          <div
            class="buttonDiv"
          >
            <button
              type="button"
            >
              4
            </button>
            <button
              type="button"
            >
              5
            </button>
            <button
              type="button"
            >
              6
            </button>
            <button
              class="operation"
              type="button"
            >
              -
            </button>
          </div>
          <div
            class="buttonDiv"
          >
            <button
              type="button"
            >
              1
            </button>
            <button
              type="button"
            >
              2
            </button>
            <button
              type="button"
            >
              3
            </button>
            <button
              class="operation"
              type="button"
            >
              +
            </button>
          </div>
          <div
            class="buttonDiv"
          >
            <button
              class="zero"
              type="button"
            >
              0
            </button>
            <button
              type="button"
            >
              .
            </button>
            <button
              class="operation"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
